admin/app/indexes/ContentApiTagsEnumerator.scala

```diff
@@ -39,7 +37,7 @@ object ContentApiTagsEnumerator extends Logging {
   }
 
   def enumerateTagType(tagType: String) = enumeratePages { page =>
-    getResponse(ContentApiClient.tags.tagType(tagType).pageSize(MaxPageSize).page(page))
+    contentApiClient.getResponse(contentApiClient.tags.tagType(tagType).pageSize(MaxPageSize).page(page))
```

Comment:
@alexduf you've probably already noticed this pattern all over the place. I found it weird but I don't want to touch the internal of ContentApiClient at this stage. I will be happy to revisit later though